src/main/java/at/hannibal2/skyhanni/test/SkyHanniConfigSearchResetCommand.kt

```diff
@@ -107,13 +114,43 @@ object SkyHanniConfigSearchResetCommand {
         val shimmy = Shimmy.makeShimmy(root, list) ?: return "§cCould not change config element '$term', not found!"
         return try {
             shimmy.setJson(element)
-            "§eChanged config element $term."
+            "§eChanged config element $term to $rawJson."
         } catch (e: Exception) {
-            ErrorManager.logError(e, "Could not change config element '$term' to '$rawJson'")
+            ErrorManager.logErrorWithData(
+                e, "Could not change config element '$term' to '$rawJson'",
+                "args" to args.toList()
+            )
             "§cCould not change config element '$term' to '$rawJson'"
         }
     }
 
+    private suspend fun toggleCommand(args: Array<String>): String {
+        if (args.size == 1 || args.size == 3) return "§c/shconfig toggle <config name> [value 1] [value 2]"
+
+        val path = args[1]
+        val rawJson1 = if (args.size > 2) args[2] else "true"
+        val rawJson2 = if (args.size > 2) args[3] else "false"
+
+        return try {
+            val (argsFilter) = createFilter(true) { path.lowercase() }
+            val (classFilter) = createFilter(false) { path.lowercase() }
+
+            val currentValue = findConfigElements(argsFilter, classFilter, true).toString()
+            val newValue = when (currentValue) {
```

Comment:
You can one line this with `val newValue = if (currentValue == "[$rawJson1]") rawJson2 else rawJson1`